### PR TITLE
[Fix] Newsfeed should be blank for newly created organizations 

### LINKED ISSUE #663: Newsfeed should be blank for newly created organizations
- **Issue Description:** Newsfeed should be blank for newly created organizations. Refer #537 
- **Issue Severity:** Medium
- **Environment Details:** Found in `talawa\lib\views\pages\newsfeed\newsfeed.dart`
- **Observed Behaviour:** Currently default data is getting loaded for newly created organizations.
- **Expected Behaviour:** It should be blank for newly created organizations
- **Steps to reproduce issue:** Create an organization and switch to the newsfeed tab.
- **Does this issue need immediate attention?** Yes
- **Are you willing to work on this issue:** Yes, I am willing to work on this issue!

### PR DESCRIPTION
**Pull request by Raunakk02**
**What kind of change does this PR introduce?**
Fixes newsfeed to be empty for newly created organization.

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No

**Summary**
Fixes #663 

**Does this PR introduce a breaking change?**
No

### CODE REVIEW

**Comment by palisadoes** [comment]
Please correct your code to pass our package analysis test. This needs to be resolved before we can review.

**Comment by Raunakk02** [comment]
> Please correct your code to pass our package analysis test. This needs to be resolved before we can review.

I have fixed the code now @palisadoes

File: lib/views/pages/organization/switch_org_page.dart [hunk 1]
```diff
@@ -77,13 +77,14 @@ class _SwitchOrganizationState extends State<SwitchOrganization> {
       _successToast("Switched to ${userOrg[isSelected]['name']}");
 
       //Kill all previous stacked screen
-      Navigator.of(context).popUntil(ModalRoute.withName("/"));
+      // Navigator.of(context).popUntil(ModalRoute.withName("/"));
```
Comment (Sagar2366) [inline lib/views/pages/organization/switch_org_page.dart:80]: instead of commenting, remove unwanted code.

**Review by Sagar2366 — CHANGES_REQUESTED** [review]
Adding new news to newsfeed is missing.

File: lib/views/pages/organization/switch_org_page.dart (hunk 1 above)
Comment (Raunakk02) [reply lib/views/pages/organization/switch_org_page.dart]: @Sagar2366 sir I have removed the commented code now.

**Comment by Raunakk02** [comment]
> Adding new news to newsfeed is missing.

I have fixed this problem now @Sagar2366 sir. Awaiting review!

**Comment by Sagar2366** [comment]
@Raunakk02 resolve conflicts on this PR.

**Comment by Raunakk02** [comment]
Resolved the conflicts @Sagar2366 sir.

**Comment by Sagar2366** [comment]
Ci check is failing @Raunakk02 have a look.

**Comment by Raunakk02** [comment]
> Ci check is failing @Raunakk02 have a look.

Working on it!

**Comment by Raunakk02** [comment]
@Sagar2366, I have resolved the failing CI check.

**Comment by palisadoes** [comment]
We will soon be enforcing a 300 line limit on Dart files. Please consider ways to reduce the sizes of the files you edit while maintaining functionality in future. PRs reviewed on or after May 15 will be rejected if this requirement is not met.

**Comment by Sagar2366** [comment]
@Raunakk02 please spend some more time and review PR properly.
Though button is there to add news feed for newly created organisation, it is not working as expected.
Please add required changes and attach video visuals to verify it.

**Comment by Raunakk02** [comment]
> @Raunakk02 please spend some more time and review PR properly.
> Though button is there to add news feed for newly created organisation, it is not working as expected.
> Please add required changes and attach video visuals to verify it.

I figured out the problem, it was due to the changes by a recently merged PR which was causing this issue. I have fixed it now and the newsfeed will load the data correctly as well as the add news feed button works correctly.

Here I am attaching the visuals recorded after the changes mentioned above:


https://user-images.githubusercontent.com/56073198/116239783-b177c100-a780-11eb-92a1-944b55c544b9.mp4

**Review by Sagar2366 — APPROVED** [review]

**Pull request merged**